In [6]:
# (1) Importing various Python libraries
import pandas as pd
import numpy as np
#import matplotlib.pyplot as plt


In [9]:
# (2) Reading and uploading data
read = pd.read_csv('/content/Netflix_data.csv')


FileNotFoundError: [Errno 2] No such file or directory: '/content/Netflix_data.csv'

In [8]:
read

NameError: name 'read' is not defined

In [5]:
#(3) checking top 5 rows of the data using .head() function

read.head()


Title        Date
0                               Almost Christmas  24/12/2024
1                              Our Little Secret  18/12/2024
2                 Twas the Text Before Christmas  18/12/2024
3  My Demon: Limited Series: Our Personal Heaven  10/12/2024
4      My Demon: Limited Series: The End of Fate  10/12/2024

In [6]:
#(4) Number of rows and columns in the dataset
read.shape

(115, 2)

In [7]:
# Number of rows in the dataset
len(read)

115

In [8]:
#(4) Number of columns in the dataset
len(read.columns)

2

In [4]:
#using the info function
read.info()

NameError: name 'read' is not defined

In [10]:
#(5) Description of the dataset
read.describe()

Title        Date
count                115         115
unique               115          48
top     Almost Christmas  21/07/2024
freq                   1          13

There are 115 entries in the dataset.
There are 115 unique movie titles in this dataset.
There are 48 unique dates associated with these movie titles.
One title appears most frequently in the dataset, showing up 13 times. Which means the highest number of movies watch in a single day occurred on this day.

In [3]:
#Using isna() function on our dataset
read.isna()

NameError: name 'read' is not defined

In [2]:
#Displaying the very first movie in our Dataset
#read[0:1]
read['Title'][0]


NameError: name 'read' is not defined

In [13]:
#Using the lambda function to find the number of times The Union  was watched
read['Title'].apply(lambda Title:'The Union' in Title).sum()

1

In [16]:
#Using the lambda function to find the number of times Master of The House  was watched
read['Title'].apply(lambda Title:'Master of the House' in Title).sum()


7

In [18]:
#Using the lambda function to find the number of times My Name  was watched
read['Title'].apply(lambda Title:'My Name' in Title).sum()

8

In [22]:
#Using the lambda function to find the number of times Post Cards Is  was watched
read['Title'].apply(lambda Title:'Postcards'in Title).sum()

6